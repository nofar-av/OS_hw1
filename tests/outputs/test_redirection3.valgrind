==28553== Memcheck, a memory error detector
==28553== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==28553== Using Valgrind-3.13.0 and LibVEX; rerun with -h for copyright info
==28553== Command: /home/omer/OS_wet1/smash
==28553== Parent PID: 28549
==28553== 
==28553== 
==28553== FILE DESCRIPTORS: 9 open at exit.
==28553== Open file descriptor 8: test_redirection3.tmp
==28553==    at 0x54EEC3E: open (open64.c:47)
==28553==    by 0x110B19: RedirectionCommand::execute() (Commands.cpp:426)
==28553==    by 0x11A30C: SmallShell::executeCommand(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >) (small_shell.cpp:171)
==28553==    by 0x118C97: main (smash.cpp:42)
==28553== 
==28553== Open file descriptor 7: test_redirection3.tmp
==28553==    at 0x54EEC3E: open (open64.c:47)
==28553==    by 0x110ABD: RedirectionCommand::execute() (Commands.cpp:418)
==28553==    by 0x11A30C: SmallShell::executeCommand(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >) (small_shell.cpp:171)
==28553==    by 0x118C97: main (smash.cpp:42)
==28553== 
==28553== Open file descriptor 6: test_redirection3.tmp
==28553==    at 0x54EEC3E: open (open64.c:47)
==28553==    by 0x110ABD: RedirectionCommand::execute() (Commands.cpp:418)
==28553==    by 0x11A30C: SmallShell::executeCommand(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >) (small_shell.cpp:171)
==28553==    by 0x118C97: main (smash.cpp:42)
==28553== 
==28553== Open file descriptor 5: test_redirection3.tmp
==28553==    at 0x54EEC3E: open (open64.c:47)
==28553==    by 0x110ABD: RedirectionCommand::execute() (Commands.cpp:418)
==28553==    by 0x11A30C: SmallShell::executeCommand(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >) (small_shell.cpp:171)
==28553==    by 0x118C97: main (smash.cpp:42)
==28553== 
==28553== Open file descriptor 4: test_redirection3.tmp
==28553==    at 0x54EEC3E: open (open64.c:47)
==28553==    by 0x110ABD: RedirectionCommand::execute() (Commands.cpp:418)
==28553==    by 0x11A30C: SmallShell::executeCommand(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >) (small_shell.cpp:171)
==28553==    by 0x118C97: main (smash.cpp:42)
==28553== 
==28553== Open file descriptor 3: /home/omer/OS_wet1/tests/outputs/test_redirection3.valgrind
==28553==    <inherited from parent>
==28553== 
==28553== Open file descriptor 2: /home/omer/OS_wet1/tests/outputs/test_redirection3.err
==28553==    <inherited from parent>
==28553== 
==28553== Open file descriptor 1: /home/omer/OS_wet1/tests/outputs/test_redirection3.out
==28553==    <inherited from parent>
==28553== 
==28553== Open file descriptor 0:
==28553==    <inherited from parent>
==28553== 
==28553== 
==28553== HEAP SUMMARY:
==28553==     in use at exit: 0 bytes in 0 blocks
==28553==   total heap usage: 192 allocs, 192 frees, 88,409 bytes allocated
==28553== 
==28553== All heap blocks were freed -- no leaks are possible
==28553== 
==28553== For counts of detected and suppressed errors, rerun with: -v
==28553== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
