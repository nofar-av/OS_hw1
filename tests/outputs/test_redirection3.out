smash> smash> smash> smash> smash> smash pid is 5955
smash pid is 5955
smash pid is 5955
smash pid is 5955
smash> smash> smash pid is 5955
smash> smash: sending SIGKILL signal to 0 jobs:
